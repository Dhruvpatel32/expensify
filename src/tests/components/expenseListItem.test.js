import React from 'react';
import Expenselist from '../../components/expenseListItems';
import {shallow} from 'enzyme';
import moment from 'moment';
const expense=
    {
        id:"1",
        description:"gum",
        amount:195,
        note:"",
        createdAt:0
    }
;
test('should render expense list',()=>{
    const wrapper=shallow(<Expenselist {...expense} />)
    expect(wrapper).toMatchSnapshot();
})