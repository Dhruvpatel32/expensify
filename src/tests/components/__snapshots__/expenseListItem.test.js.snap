// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list 1`] = `
<div>
  <Link
    to="/editexpanse/1"
  >
    <h3>
      gum
    </h3>
  </Link>
  <p>
    195
     ----
    0
  </p>
</div>
`;
